src/components/FourZeroFour.vue

```diff
@@ -332,32 +329,43 @@
 </script>
 
 <style>
-.not-found {
+.four-zero-four {
   display: flex;
   align-items: center;
   justify-content: center;
-  background:#0a2053;
+}
```

Comment:
This could be tailwind and combined with the next selector right?

Reply:
Yeah I think so? I'm not entirely sure why these were written this way, it's possible there's an SVG style scoping issue? I'd rather not dig into it in this PR tbh